### PR TITLE
[DO-NOT-MERGE] SMDDP MXNET BERT training scripts and launch commands

### PR DESCRIPTION
**Pull request by rondogency**
## Description ##
- Add Sagamaker compatible training scripts wrapper
- Add launch commands file to be easily launched in EC2/notebook
- Change all Horovod to SMDDP
- Add skip_save_states option to prevent massive write to fsx drive

**Currently we are hitting an intermittent hanging issue at startup, need to dive deep into it**

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

cc @dmlc/gluon-nlp-team

### CODE REVIEW

File: scripts/bert/launcher_mx_bert.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import os
```
Comment (karan6181) [inline scripts/bert/launcher_mx_bert.py:2]: Should we have this launcher script as python script or jupyter notebook script?

**Review by karan6181 — APPROVED** [review]
LGTM overall. Just one minor comment. Thank You!
